src/main/java/net/openhft/chronicle/core/Jvm.java

```diff
@@ -1056,10 +1061,11 @@ public static void addSignalHandler(final SignalHandler signalHandler) {
      */
     public static void safepoint() {
         if (SAFEPOINT_ENABLED)
-            if (IS_JAVA_9_PLUS)
+            if (IS_JAVA_9_PLUS) {
```

Comment:
why add opening brace? Chronicle's convention is that we don't do this for one-line ifs. If sonar doesn't like it we should turn off that rule

Reply:
I don't like it either TBH... but I generally do it on Chronicle code to fit with convention

Reply:
Generally, we write Java and not C. So we should not declare several variables on the same line, should not return an expression and should not use ifs with no curly brackets. 

This is especially important when there is an else clause which it was in this particular case.